src/components/VerticalTabs/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+interface VerticalTabProps {
+  elements: {
```

Comment:
would be a good idea to extract the interface. call it "INavTab", so you could use it at line 20